esi_ui/api/esi_api.py

```diff
@@ -147,6 +147,21 @@ def deploy_node(request, node):
     if 'ssh_keys' in kwargs:
         kwargs['config'] = instance_config.GenericConfig(ssh_keys=kwargs['ssh_keys'])
         del kwargs['ssh_keys']
+    
+    network_id = kwargs['nics'][0]['network']
+    
+    if kwargs['floatingIPOption'] == 'none':
+        kwargs['nics'] = [{'network': network_id}]
+    else:
+        connection_obj = esiclient(session=request.user.token.id)
```

Comment:
Make sure you only pass the request object to the esiclient function

Reply:
Earlier we were doing the different way so got caught up. Updated it.